### ISSUE TITLE
Wrong attributes order in comparison window

### ISSUE THREAD

**Issue by ADDISON74** [OpenMage/magento-lts#1289]
This is a known issue in Magento. The attributes order listed in product page is not the same in comparison window.  In the past I reported this issue on Stack Exchange and Magento Forum. Here are the links: 

https://magento.stackexchange.com/questions/64983/attributes-order-in-comparison-window-is-wrong

https://community.magento.com/t5/Can-Magento-do/Attributes-order-in-comparison-window-is-wrong/m-p/4137

I vote for adding this missing feature to OpenMage.

**Comment by seansan** [comment]
Wow! Thats a very old issue 2015 ... good one

btw is the "old" bugs database/reporting system from M1 still exist somewhere?

**Comment by ADDISON74** [comment]
Between 2014 - 2018 I reported a lot of issues in Magento 1 Bugtracker. Some of them were took in consideration. Unfortunately I don't think there is a backup to it. 

- I found all Magento CE packages are here + SB Samples: https://pubfiles.nexcess.net/magento/ce-packages/

- The last Magento Connect extensions can be found here: https://pluginarchive.com/magento. At a moment there were around 7000 extensions and in the end only half remain available.
